python_scripts/cross_validation_time.py

```diff
@@ -176,9 +176,9 @@
 print(f"The R2 on this single split is: {test_score:.2f}")
 
 # %% [markdown]
-# In this case, we see that our model is not magical anymore. Indeed, it
-# performs worse than just predicting the mean of the target. We can visually
-# check what we are predicting.
+# In this case, we see that our model is not magical anymore. Remember that a
+# negative $R^2$ means that the regressor performs worse than always predicting
```

Comment:
nitpick: sometimes we use `$R^2$` while other times we use `R²`. We should probably try to stay consistent, at least within a single md file.